docs/doc/index.md

```diff
@@ -1,19 +1,20 @@
 # RS-Server
 
-RS-Server is a comprehensive toolbox designed to facilitate the retrieval of external data for Copernicus processing
-chains. It allows users to store data into object storages and index it into STAC compatible catalogs.
+The Reference System server provides a set of services necessary to build Copernicus processing workflows. All services are subject to access control.
 
-The primary objective of RS-Server is to support Copernicus processing chains in executing their tasks efficiently.
+RS-Server offers a catalog of Sentinel products compatible with the STAC (SpatioTemporal Asset Catalog) standard, as well as functions to access data from AUXIP and CADIP stations.
 
 ## Features
 
-The toolbox provides REST endpoints that enable users to:
+To achieve this, RS-Server exposes REST endpoints that allow users to:
+* Search CADU chunks from CADIP stations
+* Stage CADU chunks from CADIP stations
+* Search auxilliary data from AUXIP station
+* Stage auxilliary data from AUXIP station
+* ALL endpoints from [SpatioTemporal Asset Catalog API](https://stacspec.org/) 
+
+Please note that the STAC catalog will also embedds STAC extension to support Sentinel product format from [EOPF-CPM](https://cpm.pages.eopf.copernicus.eu/eopf-cpm/main/index.html).
```

Comment:
* will also embedds -> it's future, the verb should be used with the base form of the verb -> will also embed

Reply:
Fine for me. Your english is better than mine. So fill free to fix any wording.